src/main/java/no/digipost/api/client/SenderId.java

```diff
@@ -26,4 +26,15 @@ public SenderId(final long id) {
     public long getId() {
         return id;
     }
+
+    public String asString() {
+        return String.valueOf(id);
+    }
+
+    @Override
+    public String toString() {
+        return "SenderId{" +
```

Comment:
I though it was better to just use the autogenerated style because it makes it easier to maintain (just regenerate). But now that the classes have stabilized maybe it should be "hand coded" after all 😄